spec/telephone_appointments/response_spec.rb

```diff
@@ -0,0 +1,41 @@
+require 'spec_helper'
+
+RSpec.describe TelephoneAppointments::Response do
+  subject { described_class.new(response) }
+
+  context 'when the response is valid' do
+    let(:response) { Net::HTTPSuccess.new(nil, nil, nil) }
+
+    it 'should be a success' do
```

Comment:
Use the unequivocal style for spec descriptions. `is a success` reads better here. Same elsewhere in this spec file.